app/src/main/java/ch/epfl/sdp/musiconnect/ProfileModification.java

```diff
@@ -0,0 +1,92 @@
+package ch.epfl.sdp.musiconnect;
+
+import androidx.appcompat.app.AppCompatActivity;
+import ch.epfl.sdp.R;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+public class ProfileModification extends AppCompatActivity implements View.OnClickListener {
+
+    String firstName, lastName, username, mail, birthday;
+    EditText[] editFields;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_profile_modification);
+
+        editFields = new EditText[] {
+                findViewById(R.id.newFirstName),
+                findViewById(R.id.newLastName),
+                findViewById(R.id.newUsername),
+                findViewById(R.id.newEmailAddress),
+                findViewById(R.id.newBirthday)};
+
+        firstName = getIntent().getStringExtra("FIRST_NAME");
```

Comment:
It's a good practice to have static string variables for the Strings used to pass extras in intents.